data/pc/latest/proto.yml

```diff
@@ -2623,16 +2623,18 @@
       # ]]
       flags: u8
    # MC: ClientboundUpdateRecipesPacket
-   # TODO: update packet
    packet_declare_recipes:
-      recipes: []varint
-         name: string
+      propertySetCount: varint
```

Comment:
why? `properySets: []varint` already reads the length of the array

Reply:
I was just following the protocol shown [here](https://wiki.vg/Pre-release_protocol#Update_Recipes), if its not needed then `propertySetCount` could probably be removed.

Reply:
In `properySets: []varint` the varint means that the size of this array is given by a varint field preceding the array.